frontera/contrib/backends/memory/__init__.py

```diff
@@ -128,11 +135,11 @@ def _get(self, obj):
 
     def update_cache(self, objs):
         objs = objs if type(objs) in [list, tuple] else [objs]
-        map(self._put, objs)
+        list(map(self._put, objs))
```

Comment:
Why list is needed here? The idea is to apply self._put to every element in the objs